biggraphite/drivers/elasticsearch.py

```diff
@@ -105,6 +171,34 @@ def add_argparse_arguments(parser):
     )
 
 
+def _components_from_name(metric_name):
+    res = metric_name.split(".")
+    return list(filter(None, res))
+
+
+def document_from_metric(metric):
+    """Creates an ElasticSearch document from a Metric."""
+    config = metric.metadata.as_string_dict()
+    components = _components_from_name(metric.name)
+    name = ".".join(components)  # We do that to avoid double dots.
+
+    data = {
+        "length": len(components),
```

Comment:
not true..

Reply:
I propose it merge that as is, and then we change that from length to component